assignments/assignments_parser.go

```diff
@@ -55,6 +54,7 @@ func parseAssignments(dir string, courseID uint64) ([]*pb.Assignment, error) {
 				if err != nil {
 					return fmt.Errorf("could not to read %q file: %w", filename, err)
 				}
+				labDirectoryName := filepath.Base(filepath.Dir(path))
```

Comment:
No need to declare this as a separate variable, just do it inline in the creation of the `pb.Assignment` object on line 74. But also add a comment right above Line 70:
```
// The assignment Name field is the folder name of the assignment.
```
And also remove the TODO at the top of the func, since this solves the problem.

Reply:
Done.